client/jetpack-onboarding/main.jsx

```diff
@@ -128,12 +140,15 @@ export default connect(
 			STEPS.STATS,
 			STEPS.SUMMARY,
 		] );
+		const stepsCompleted = getJetpackOnboardingCompletedSteps( state, siteId, steps );
+
 		return {
 			isRequestingSettings,
 			siteId,
 			siteSlug,
 			settings,
 			steps,
+			stepsCompleted,
```

Comment:
Maybe we should start inlining consts like `stepsCompleted` here in order to cut back a few lines of `connect`.